compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/KotlinPoetUtils.kt

```diff
@@ -25,7 +25,7 @@ import java.io.ByteArrayOutputStream
 public fun ClassDescriptor.asClassName(): ClassName =
   ClassName(
     packageName = parents.filterIsInstance<PackageFragmentDescriptor>().first()
-      .fqName.safePackageString(),
+      .fqName.safePackageString(dotSuffix = false),
```

Comment:
Curious, did a unit test fail by any chance without this change?

Reply:
Yes, `AssistedFactoryGeneratorTest#the factory function may be provided by a generic super type from another module` failed without the fix